docs/articles_en/documentation/openvino_ir/operation_sets/operations_specifications/activation/ReLU_1.rst

```diff
@@ -5,14 +5,14 @@ ReLU
 
 
 .. meta::
-  :description: Learn about ReLU-1 - an element-wise, activation operation, which 
+  :description: Learn about ReLU-1 - an element-wise, activation operation, which
                 can be performed on a single tensor in OpenVINO.
 
 **Versioned name**: *ReLU-1*
 
 **Category**: *Activation function*
 
-**Short description**: ReLU element-wise activation function. (`Reference <http://caffe.berkeleyvision.org/tutorial/layers/relu.html>`__).
+**Short description**: ReLU element-wise activation function.
```

Comment:
this does not relate to OpenVINO supporting the framework, there's no need to remove it.

Reply:
Change applied in https://github.com/openvinotoolkit/openvino/pull/22509/commits/ef18601b07d035c40555cb80978cca7d3fb76669